setup.cfg

```diff
@@ -1,6 +1,9 @@
 [metadata]
 description-file = README.md
 
+[bandit]
+skip = B404
```

Comment:
does this actually work

Reply:
It did complain about B404, so I don't think so.